docs/extensions/attributetable.py

```diff
@@ -153,7 +153,7 @@ def run(self) -> list[AttributeTablePlaceholder]:
 def build_lookup_table(env: BuildEnvironment) -> dict[str, list[str]]:
     # Given an environment, load up a lookup table of
     # full-class-name: objects
-    result = {}
+    result: dict[str, list[str]] = {}
```

Comment:
But for these ones it's interesting to have type checking

Reply:
I agree, in some cases, being forced to explicitly declare the annotations can be very helpful to the type checker, it is nice that this would now be a requirement, where types can't be easily inferred.